.github/workflows/prepare-version-bump.yml

```diff
@@ -0,0 +1,50 @@
+name: Autobump version on PR
+
+on:
+  pull_request:
+    branches:
+    - main
+
+jobs:
+  prepare-new-version:
+    environment: Autobump version
+    runs-on: ubuntu-latest
+    if: ${{ github.repository == 'Cardinal-Cryptography/AlephBFT'}}
+    steps:
+      - name: cargo-next-install
+        run: |
+          cargo install cargo-next --locked
+      - name: checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 2
+          ref: ${{ github.head_ref }}
+          token: ${{ secrets.SYNCAZF || secrets.MY_TOKEN || github.token }}
+      - name: check-and-bump
+        run: |
```

Comment:
nit: this block can be made into sh script (and put somewhere in the `scripts/...` folder). This way it will be more readable here and easier to review the script without the noise from the workflow file.